### DISCUSSION TITLE
How to achieve Github Profile Badge ?

### DISCUSSION THREAD

**Discussion by Chanodyapraveen** [community/community#72206] [New to GitHub]
Hi I am Chanodya Praveeen Thapassarage.
Undergraduate at NSBM Green University.

**Comment by Isuru-Eranda** [comment]
Study this Github repository
Link - https://github.com/Thinkright20/Profile-Badges

**Reply by Chanodyapraveen** [reply]
Thanks for your  answer

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
